Plots/Trajectory/NCL_polyg_14.py

```diff
@@ -0,0 +1,85 @@
+
+###############################################################################
+# Import packages:
+import numpy as np
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+from geographiclib.geodesic import Geodesic
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+###############################################
+
+def Plot(color, row, col, pos, title, style, pt):
+
+    fig = plt.figure(figsize=(12,12))
+    ax = fig.add_subplot(row, col, pos, projection=ccrs.PlateCarree())
+    
+    ax.set_extent([-125,-60,15,65], ccrs.PlateCarree())
+    ax.add_feature(cfeature.LAND, color='lightgrey')
+    
+    plt.plot([-120, -64], [20, 60], style, color= color,  transform=ccrs.Geodetic())
+    
+    # start location
+    lon_fr = -120
+    lat_fr = 20
+    # end location
+    lon_to = -64
+    lat_to = 60
+    
+    # This gets geodesic between the two points
+    # WGS84 ellipsoid is used
+    gl = Geodesic.WGS84.InverseLine(lat_fr, lon_fr, lat_to, lon_to)
+    
+    num_points = 10  #for points on geodesic
+    print("distance latitude longitude azimuth")
+    
+    # Compute points on the geodesic, and plot them as red dots
+    # gl.s13 is the total length of the geodesic
+    # the points are equally spaced by true distances, but not on the map
+    # due to the projection distortion
+    for ea in np.linspace(0, gl.s13, num_points):
+        g = gl.Position(ea, Geodesic.STANDARD | Geodesic.LONG_UNROLL)
```

Comment:
I don't see any functionality in cartopy that can easily do what you're trying to do here. We may have to add `geographiclib` to the environment if there is no work around.
cc: @clyne

Reply:
geographiclib is distributed via conda. So I don't see any problem with making use of it. Checking with @erogluorhan

Reply:
I agree with @clyne as long as @michaelavs adds geographiclib as a dependency to the conda environment yml file within this PR.

Reply:
@erogluorhan should I just push the conda_environment.yml with the geogrpahiclib dependency to the newer PR of this one? Or do I need to create a new PR for changing the yml?

Reply:
Yes, it would be good enough to push the `conda_environment.yml` with the `geographiclib` dependency to the newer PR of this one without a need of creating another new PR.